scene/2d/sprite.cpp

```diff
@@ -135,12 +135,12 @@ void Sprite::set_texture(const Ref<Texture> &p_texture) {
 		return;
 
 	if (texture.is_valid())
-		texture->remove_change_receptor(this);
+		texture->disconnect(CoreStringNames::get_singleton()->changed, this, "_texture_changed");
```

Comment:
This is good, because change receptor is only meant to be used by the editor. It needs to remain but the function will change for 4.0. If you see this used in normal scene nodes, its good to change it like here.